docs/en/02_Developer_Guides/05_Extending/05_Injector.md

```diff
@@ -111,6 +111,16 @@ Now the dependencies will be replaced with our configuration.
 	echo ($object->textProperty == 'My Text Value');
 	// returns true;
 
+As well as properties, method calls can also be specified:
+
+	:::yml
+	Injector:
+	  Logger:
+	    class: Monolog\Logger
+	    methods:
+	      - [ pushHandler: [ %$DefaultHandler ] ]
```

Comment:
I think it's meant to be a two-length array, I mean, not a one-length associative array?